lerobot/scripts/eval.py

```diff
@@ -194,6 +195,7 @@ def rollout(
         "reward": torch.stack(all_rewards, dim=1),
         "success": torch.stack(all_successes, dim=1),
         "done": torch.stack(all_dones, dim=1),
+        "observation": None,
```

Comment:
it lowers the cognitive load by knowing exactly what could be returned by this rollout function, instead of having to hunt it down by reading the entire function first. That's why I always favor "strongly" typed return objects (e.g. dataclasses, pydantic models, protobuf, etc...).

But this is a preference as you said, more than happy to revert it.